app/assets/javascripts/details/shell/action_bar_bridge.js

```diff
@@ -0,0 +1,116 @@
+var ActionBarBridge;
```

Comment:
@dogweather I agree with the window/global overkill. I was doing this intentionally to make these files testable while running the beta testing. The scope can be removed, but it helps with debugging.

These files will eventually be in the asset pipeline. When we make that switch, the plan is to change how the files are invoked.